invenio.cfg

```diff
@@ -312,6 +316,24 @@ CDS_REDIRECTION_COLLECTIONS_MAPPING = {
     "CERN Students Projects": "",  # uuid
 }
 
+RDM_SEARCH = {
+    **deepcopy(RDM_SEARCH),
+    "query_parser_cls": QueryParser.factory(
+        mapping={
```

Comment:
This is currently an issue, also happening in Zenodo [here](https://github.com/zenodo/zenodo-rdm/blob/master/invenio.cfg#L550).
We have recently introduced protected internal fields [here](https://github.com/inveniosoftware/invenio-rdm-records/blob/c807b72a4e62054df3c90cbdbe36c9a305c962dc/invenio_rdm_records/config.py#L277), but when overriding the config variable, this is gone. This should be changed, and made internal, not overridable.